components/raft_log_engine/Cargo.toml

```diff
@@ -19,5 +19,5 @@ serde = "1.0"
 serde_derive = "1.0"
 kvproto = { git = "https://github.com/pingcap/kvproto.git" }
 raft = { version = "0.7.0", default-features = false, features = ["protobuf-codec"] }
-raft-engine = "0.1.0"
+raft-engine = { git = "https://github.com/tikv/raft-engine.git" }
```

Comment:
How about bumping a patch version?

Reply:
I think we should use latest raft engine to catch bugs early. The patch version will be release in sync with 6.1.

Reply:
This will be slow down compilation. One goal of tirocks is to skip git pulling for every fresh build.